dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -107,8 +109,74 @@
 
     private transient ContentletAPI contentletAPI;
     private transient UserAPI userAPI;
+	private transient WriteRequest.RefreshPolicy refreshPolicy;
+
+	public Object getRefreshPolicy () {
```

Comment:
I get that we are trying to reuse Elastic's enum without being tied to it but I think we need our own enum `IndexPolicy` and rename this `public IndexPolicy getIndexPolicy() ` . and return our enum here, e.g.

**IndexPolicy**
- indexLater
- waitForIndex
- forceIndex

Reply:
maybe `indexAfter` or `deferIndex` instead of `indexLater`

Reply:
ok I will change it